DevCycle.SDK.Server.Common/API/DevCycleBaseClient.cs

```diff
@@ -13,7 +13,7 @@ namespace DevCycle.SDK.Server.Common.API
 {
     public abstract class DevCycleBaseClient : IDevCycleClient 
     {
-        public string SdkPlatform => $"C# {Platform()}";
+        public string SdkPlatform => $"C#";
```

Comment:
Should we be using `C#` or `.NET` here? On the dashboard it was trying to target `.NET Local` before I fixed it.

Reply:
C#